src/main/java/br/com/freeanki/utils/exceptions/ResourceExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package br.com.freeanki.utils.exceptions;
+
+import br.com.freeanki.services.exception.ObjectNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import javax.servlet.http.HttpServletRequest;
+
+@ControllerAdvice
+public class ResourceExceptionHandler {
+
+    @ExceptionHandler(ObjectNotFoundException.class)
+    public ResponseEntity<StandardError> objectNotFound(ObjectNotFoundException e, HttpServletRequest request) {
+        StandardError error = new StandardError(HttpStatus.NOT_FOUND.value(), "Object not found", e.getMessage(), System.currentTimeMillis(), request.getRequestURI());
```

Comment:
- Linha muito grande, ctrl + alt + L
- extrair "Object not found" para uma constante estatica